README.adoc

```diff
@@ -147,6 +147,11 @@ If set to `false`, the binder relies on the partition size of the topic being al
 If the partition count of the target topic is smaller than the expected value, the binder fails to start.
 +
 Default: `false`.
+spring.cloud.stream.kafka.binder.autoAlterTopics::
+If set to `true`, the binder alters destination topic configs if required.
```

Comment:
We should probably add a comment that the broker version must be at least 2.3 to use this feature.